Public/Src/FrontEnd/Core/FrontEndHostController.cs

```diff
@@ -112,7 +112,7 @@ public new Workspace Workspace
 
         private readonly PerformanceCollector m_collector;
 
-        private static readonly TimeSpan EvaluationProgressReportingPeriod = TimeSpan.FromMilliseconds(500);
+        private static readonly TimeSpan EvaluationProgressReportingPeriod = TimeSpan.FromMilliseconds(1000);

```

Comment:
Can we use `BuildXLEngine.GetTimerUpdatePeriodInMs` here for consistency across all other updating messages like this?
https://github.com/microsoft/BuildXL/blob/cc80caf8f8556801eb34ea8f90452067848b48fc/Public/Src/Engine/Dll/Engine.cs#L3335

Reply:
The evaluation usually takes a few seconds(10ish from a few builds I looked at) to finish and the GetTimerUpdatePeriodInMs is 10 secs now. So if we use GetTimerUpdatePeriodInMs, we won't actually get the update messages of the evaluation progress.

Reply:
I think that's ok to for there not to be frequent status updates for sub 10 second evaluations. The idea of those console status updates is to communicate details when something very long running so the user doesn't think the app is hung.